### PR TITLE
Env config

### PR DESCRIPTION
**Pull request by philipep-galdino**
# PR Set up env config 

_Set up .env configs and the configuration service to use .env in NestJS with TypeOrm connections options for default connection_

## How should this be tested?

1. Run up the application w/ database set up
2. Run up yarn typeorm schema:sync

## Notes or observations

_As I was having troubls w setting up a .env simply, because for some reason nestjs implies agaisnt that and tells you to use their configuration service, I did as told by their documentation and examples on standard moves with that. Basically now there is a whole configuration service running for the database and the ORM configs. 

Note that I also left the old config.ts file as I didn't want to set up another configuration service only for two simple variables which we're not even using right now as it goes - the ones for the Indexer_

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/129

### CODE REVIEW

**Comment by philipep-galdino** [comment]
@kesar 👋🏾 

`$ ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js schema:sync
Error during schema synchronization:
TypeORMError: No connection options were found in any orm configuration files.
`

This is what happens when i try to run schema:sync. It simply doesn't recognize the ormconfig from the database.config.ts file. Weirdly, not even when I set up the directory manually for that, as you can see below: 

`$ ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli.js --config ./src/Database/database.config.ts schema:sync
Error during schema synchronization:
TypeORMError: Cannot find connection default because its not defined in any orm configuration files.
`
I've seen a lot of other examples of people setting it up just like that and it was working, I've dug this for too much time now 😅

File: src/config.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
-// TODO: get from .envs
-export default {
+const config = {
```
Comment (kesar) [inline src/config.ts:1]: this file should be fully removed if we move to the other config approach

**Comment by philipep-galdino** [comment]
done! check my comment on the last commit hehe and we'll be ready to merge this too

**Comment by philipep-galdino** [comment]
alright, now it's really done. 🤣

**Review by kesar — APPROVED** [review]

**Comment by kesar** [comment]
this got merged but .env was not created on dev server

**Pull request merged**